doc/thing/supported-boards/knot-dongle.rst

```diff
@@ -0,0 +1,125 @@
+KNoT Dongle
+===========
+
+Overview
+--------
+
+The KNoT Dongle (nRF52840-PCA10059) is a low-cost board manufactured by Nordic
+Semiconductor and it has a user programmable RGB LED.
+
+.. figure:: ../../../_static/dongle.png
+   :scale: 80 %
+   :alt: Dongle - nRF52840 (PCA10059)
+   :align: center
+
+   Dongle - nRF52840 (PCA10059)
+
+More information about the board can be found at the `nRF52840 Dongle website
+<https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-Dongle>`_
+and `Zephyr nRF52840-PCA10059 website
+<https://docs.zephyrproject.org/latest/boards/arm/nrf52840_pca10059/doc/index.html>`_.
+The `Nordic Semiconductor Infocenter
+<https://infocenter.nordicsemi.com/topic/ug_nrf52840_dongle/UG/nrf52840_Dongle/intro.html>`_
+contains the processor's information and the datasheet.
+
+-------------------------------------------------------------------------------
+
+Supported Features
+------------------
+
+The Dongle hardware provides support for the following devices:
+
+   - ADC
+   - CLOCK
+   - FLASH
+   - GPIO
+   - I2C
+   - MPU
+   - NVIC
+   - PWM
+   - RTC
+   - SPI
+   - UART
+   - USB
+   - WDT
+
+-------------------------------------------------------------------------------
+
+Connections and IOs
+-------------------
+
+.. figure:: ../../../_static/dongle_buttons_leds.png
```

Comment:
Why pointing `Button (SW1)` when it's already described as `SW1`?

Reply:
I agree it is not necessary, but the image is from Nordic Semiconductor and it is available this way on their website.

Reply:
Use this one: https://infocenter.nordicsemi.com/topic/ug_nrf52840_dongle/UG/nrf52840_Dongle/Images/nRF52840_dongle_front.svg

Reply:
Done.